cluster/cluster.go

```diff
@@ -70,6 +72,14 @@ func (c *Cluster) DeployClusterPlanes() error {
 	if err != nil {
 		return fmt.Errorf("[workerPlane] Failed to bring up Worker Plane: %v", err)
 	}
+	if err = authz.ApplyJobExecuterServiceAccount(c.LocalKubeConfigPath); err != nil {
```

Comment:
I think this should happen after deploying control plane directly and not in after worker plane to avoid kubelet to generate error logs